packages/web/src/services/Location.ts

```diff
@@ -21,10 +21,24 @@ export type Location = {
   utc_offset: string
 }
 
+type CachedLocation = {
+  location: Location
+  expires: number
+}
+const CACHED_LOCATION_TTL_MS = 5000
+let cachedLocation: CachedLocation | null = null
```

Comment:
idk if this is a me thing or not, but i like

const cachedLocation = {
  loc: null
}

cachedLocation.null = X


just because mutable things scare me to some degree b/c you might not know where it's coming from.

just personal pref tho, pls ignore me unless you decide you really agree